config/initializers/assets.rb

```diff
@@ -26,6 +26,4 @@
 Rails.application.config.assets.precompile += %w( potlatch2/FontLibrary.swf )
 Rails.application.config.assets.precompile += %w( potlatch2/locales/*.swf )
 Rails.application.config.assets.precompile += %w( help/introduction.* )
-Rails.application.config.assets.precompile += %w( iD/img/*.svg iD/img/*.png iD/img/*.gif )
-Rails.application.config.assets.precompile += %w( iD/img/pattern/*.png )
-Rails.application.config.assets.precompile += %w( iD/locales/*.json )
```

Comment:
What's this all about? Why not just list the files in the normal way, and how does that new code actually work?